Automattic/woocommerce-payments
# Return to cart page when coming back from WooPay

#### Changes proposed in this Pull Request









- Sets the `return_url` when initializing the WooPay session to the store's cart page.
- Removes the `return_url` parameter passed to the WooPay init function.

Discussions in p1673911142216149-slack-C022WMN88KG.

#### Testing instructions





* Initialize a WooPay session from the Checkout, Cart, and Product pages.
* For every WooPay session make sure the "Back" button brings you back to the Store's cart page with the correct items in the cart.

-------------------


**Post merge**
